packages/fx-core/templates/plugins/resource/apiconnector/sample/ts/aad.template

```diff
@@ -0,0 +1,18 @@
+import {
+  TeamsFx,
+  createApiClient,
+  BearerTokenAuthProvider,
+  IdentityType,
+} from "@microsoft/teamsfx";
+
+// Loads current app's configuration and use app for auth.
+const teamsFx = new TeamsFx(IdentityType.App);
+// Initializes a new axios instance to call {{config.APIName}} API
```

Comment:
also support cases that users want to use their own AAD app

Reply:
I will add it in next PR.